permissioned-token/sdk/.solitarc.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const programDir = path.join(__dirname, '..', 'program');
+const idlDir = path.join(__dirname, 'idl');
+const sdkDir = path.join(__dirname, 'src', 'generated');
+const binaryInstallDir = path.join(__dirname, '.crates');
```

Comment:
We did this for compression, and I thought it was a nice idea to avoid bloating in random directories -- how about putting the binaries in `target/solita`?

Reply:
Could you do this one to avoid accidentally checking in the `.crates` folder in the future?

Reply:
done